src/lib/noteServices.js

```diff
@@ -5,27 +5,27 @@ export const getNotes = async () => {
   return res.json();
 };
 
-export const createNote = async title => {
+export const createNote = async content => {
```

Comment:
this had to change to support the way draftjs wants the editor state saved on the server